### PR TITLE
Fixed misaligned text in action dropdown

### LINKED ISSUE #8957: Datagrid: Text is misaligned in action dropdown
**Describe the bug**
Text icon in action dropdown is misaligned after selection 

**To Reproduce**

1. Go to : 'https://main-enterprise.demo.design.infor.com/components/datagrid/test-cascading-dropdowns.html' 
2.  Select item in Action dropdown  

**Expected behavior**
 text should be in middle 

**Version**

-ids-enterprise-wc: 1.4.1
Screenshot 2024-08-09 023236

### PR DESCRIPTION
**Pull request by rishang14**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
 
Fixed  misaligned  text in action dropdown

**Related github/jira issue (required)**:
 
Fixes #8957 

**Steps necessary to review your pull request (required)**:

**Included in this Pull Request**:

- [✔️ ] A note to the change log.

### CODE REVIEW

**Review by jbrcna — CHANGES_REQUESTED** [review]
image

in firefox, the text is not centered
image

**Review by glenlieorillo — CHANGES_REQUESTED** [review]
The alignment still needs to adjust to the center in Mac Firefox.

Mac - Firefox 
image

Mac - Chrome
image

Mac - Edge
image

Safari
image

File: src/components/dropdown/_dropdown.scss [hunk 1]
```diff
@@ -102,7 +102,7 @@ div.multiselect {
     display: inline-block;
     overflow: hidden;
     text-overflow: ellipsis;
-    padding: 7px 0;
+    // padding: 7px 0;
```
Comment (tmcconechy) [inline src/components/dropdown/_dropdown.scss:105]: Also we prefer to remove code rather than leave it in commented out?

**Comment by tmcconechy** [comment]
@rishang14 be sure to test:

http://localhost:4000/components/dropdown/example-index.html
http://localhost:4000/components/multiselect/example-index.html

As i think this is effecting there. Might need this fix isolated in datagrid.scss

**Comment by tmcconechy** [comment]
@rishang14 can you follow up with some of the updates suggested and fix the lint issues? thanks for your contribution!

**Review by tmcconechy — DISMISSED** [review]

**Review by ericangeles — APPROVED** [review]

**Review by jbrcna — APPROVED** [review]
image

**Pull request merged**